src/Symfony/Component/Serializer/DataCollector/SerializerDataCollector.php

```diff
@@ -21,7 +21,7 @@
 /**
  * @author Mathias Arlaud <mathias.arlaud@gmail.com>
  *
- * @internal
+ * @final
```

Comment:
```suggestion
 * @final since 6.4
```
?

Do we plan to make it real final?